### ISSUE TITLE
Unable to sign into SVL Simulator

### ISSUE THREAD

**Issue by darshanmat** [lgsvl/simulator#2080]
It gives an error at the right bottom of the page saying **_something is wrong please try later!_**

**Comment by darshanmat** [comment]
Tried resetting the password and yet I get the same error

**Comment by knnurl** [comment]
+++, not logging in from any device for a while now.

**Comment by okuzucu20** [comment]
I am also facing with the same issue

**Comment by mslavescu** [comment]
Check this issue for a fork of LG SVL which doesn't require wise cloud:
https://github.com/lgsvl/simulator/issues/1994

**Comment by WZhijun154** [comment]
help!

**Comment by JasonBourne1998** [comment]
I guess you could use #2071 to coutinue using SVL

**Comment by YuqiHuai** [comment]
Yes. I collected all the publicly available assets and built this local WISE. You can keep your existing projects going if I was able to collect your sensor configuration.

OSSDC is actually modifying the client so improvements can be made. Right now both approaches lack ability to add private asset. I’m also working on this feature for the local cloud.

**Comment by darshanmat** [comment]
I am able to sign in now(via cloud)

**Comment by mslavescu** [comment]
Check this comment on how to use other maps and vehicles configs in OSSDC SIM:

https://github.com/OSSDC/OSSDC-SIM/issues/3#issuecomment-1166593428

**Comment by zhanghe1876168** [comment]
> It gives an error at the right bottom of the page saying **_something is wrong please try later!_**

hello,have you solved this problem？
